assets/src/edit-story/output/page.js

```diff
@@ -95,7 +99,11 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
           </amp-story-grid-layer>
         )}
 
-        <amp-story-grid-layer template="vertical" aspect-ratio={ASPECT_RATIO}>
+        <amp-story-grid-layer
+          id="amp-story-grid-layer-fullbleed-area"
```

Comment:
Ditto, should use `class` instead of `id`

Reply:
Got it.